spec.bs

```diff
@@ -491,7 +494,7 @@ A [=component=] has an associated <dfn for=component>has regexp groups</dfn>, a
 </div>
 
 <div algorithm>
-  To perform a <dfn export for=URLPattern>match</dfn> given a {{URLPattern}} |urlpattern|, a {{URLPatternInput}} |input|, and an optional string |baseURLString|:
+  To perform a <dfn export for=URLPattern>match</dfn> given a [=URL Pattern=] |urlpattern|, a {{URLPatternInput}} |input|, and an optional string |baseURLString|:
```

Comment:
Inside _match_, we still rely on some objects. In step 11, [URLPatternInit](https://whatpr.org/urlpattern/219.html#dictdef-urlpatterninit), in step 22, [URLPatternResult](https://whatpr.org/urlpattern/219.html#dictdef-urlpatternresult). Maybe we have to define them as structs as well?

Reply:
I am not 100% confident, but I guess it fine for inputs but may need struct for output?
Note that URLPatternResult seems not be recorded to realm.  I would like to know @domenic thought on this.

Reply:
URLPatternInit and URLPatternResult aren't JavaScript objects at this point. The WebIDL layer converts them from JavaScript values into [Infra maps](https://infra.spec.whatwg.org/#maps), which are not related to any particular realm (in the ECMAScript sense).

Reply:
Do you mean I do not need to define structs that represents `URLPatternInit` or `URLPatternResult`?
If so, let me mark this thread resolved.

Reply:
I think so, thanks Jeremy for clarification.

Reply:
Thank you both!
Let me mark this resolved then.